lib/gulp-tasks/file-globbing.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
@sergesemashko I see the point, as globbing is a relatively unknown term, but it is the *technical* term. And if a person was to search for a script that does this type of functionality, they would have much better luck using `globbing` or `glob` than they would if they used `inject`.

In node, the primary module which does this, which all others seem to be build upon, is called [node-glob](https://github.com/isaacs/node-glob). I'd like to keep this as `file-globbing`

@beynya debate?